tests/core/test_nlg.py

```diff
@@ -328,3 +331,271 @@ def test_nlg_fill_response_quick_replies(
         filled_slots={quick_replies_slot_name: quick_replies_slot_value},
     )
     assert result == {"quick_replies": str(quick_replies_slot_value)}
+
```

Comment:
can we at least move your tests to a module `tests.core.nlg.test_response.py` to mirror that you're testing the `response.py` module? It's legacy that we don't have this mirroring between production code and test code and it's not your job to fix it but we can at least do it right for new tests. This way it's way easier to see which tests are testing what - otherwise one might have to search for tests.

Reply:
Yup sure, two questions:
- Should I move both the new `test_response.py` and the legacy `test_nlg.py` to `tests.core.nlg` directory?
- I won't have to rename all tests in `test_response.py` from `test_nlg` to `test_response_etc`, would I? 😬

Reply:
> Should I move both the new test_response.py and the legacy test_nlg.py to tests.core.nlg directory?

Up to you. In my opinion it's also fine to focus upon your changes and to keep the diff small.